src/backend/dx12ll/src/lib.rs

```diff
@@ -361,14 +361,14 @@ impl core::Surface for Surface {
     }
 }
 
-pub struct SwapChain {
-    inner: ComPtr<winapi::IDXGISwapChain3>,
+pub struct Swapchain {
+    inner: ComPtr<winapi::IDXGISwapchain3>,
```

Comment:
It's hard to trace these down without a working Windows environment, so I opened the PR to see which ones need fixing.